src/Validation/DNSRecords.php

```diff
@@ -8,12 +8,12 @@ class DNSRecords
     /**
      * @var array $records
      */
-    private $records = [];
+    private $records;
 
     /**
      * @var bool $error
      */
-    private $error = false;
+    private $error;
```

Comment:
Please keep it as boolean false, rather than null.

Reply:
The initializer is redundant as the constructor immediately overwrites it.
We can either drop the redundant initializer or to make things more explicit, we could make this `private array $records` and drop the comment above which will also be redundant then.

Reply:
Yep, good point. Let's go with the type.